support/ebpf/bpfdefs.h

```diff
@@ -103,39 +105,40 @@ static long (*bpf_probe_read_kernel)(void *dst, int size, const void *unsafe_ptr
       bpf_trace_printk(____fmt, sizeof(____fmt), ##__VA_ARGS__);                                   \
     })
 
-  #ifdef OPTI_DEBUG
-    #define DEBUG_PRINT(fmt, ...) printt(fmt, ##__VA_ARGS__);
-
-    // Sends `SIGTRAP` to the current task, killing it and capturing a coredump.
-    //
-    // Only use this in code paths that you expect to be hit by a very specific process that you
-    // intend to debug. Placing it into frequently taken code paths might otherwise take down
-    // important system processes like sshd or your window manager. For frequently taken cases,
-    // prefer using the `DEBUG_CAPTURE_COREDUMP_IF_TGID` macro.
-    //
-    // This macro requires linking against kernel headers >= 5.6.
-    #define DEBUG_CAPTURE_COREDUMP()                                                               \
-      ({                                                                                           \
+  #define DEBUG_PRINT(fmt, ...)                                                                    \
```

Comment:
The macros DEBUG_PRINT, DEBUG_CAPTURE_COREDUMP and DEBUG_CAPTURE_COREDUMP_IF_TGID can now be converted into `static inline` functions.

Reply:
If there are no further concerns, I would like to keep these macros. With this macros it is easier to spot them and assign their use case.

Reply:
I don't have "concerns", just suggesting to follow good engineering practice (search for "static inline vs macros").
If your concern is "With this macros it is easier to spot them", why not keep the same uppercase names as function names?

Reply:
There are no variadic functions in eBPF AFAIK.

Reply:
I think conversion to function is not possible in this case. The `if` needs to be first and inlined. Otherwise the compiler might calculate the arguments even if logging is disabled. I'd prefer to keep these as defines.